src/app/static/src/app/components/projects/ShareProject.vue

```diff
@@ -17,6 +17,7 @@
                                class="form-control"
                                :class="{'is-invalid': email.valid === false}"
                                @blur="() => addEmail(email, index)"
+                               @mouseout="$event.target.blur()"
```

Comment:
This is a good solution. One potential downside is the user might get confused about why they are no longer selecting the input if the mouse moves out of it for whatever reason, which isn't typical behaviour when browsing the web. I still think this is the best solution proposed so far though.

Reply:
I will create a ticket to look into this. thanks...